users/events/streams.go

```diff
@@ -14,7 +14,20 @@ import (
 	"github.com/go-chi/chi/v5/middleware"
 )
 
-const streamID = "supermq.users"
+const (
+	supermqPrefix      = "supermq."
+	createStream       = supermqPrefix + userCreate
+	updateStream       = supermqPrefix + userUpdate
+	changeStatusStream = supermqPrefix + changeUserStatus
+	viewStream         = supermqPrefix + userView
+	listStream         = supermqPrefix + userList
+	searchStream       = supermqPrefix + userSearch
+	identifyStream     = supermqPrefix + userIdentify
+	tokenStream        = supermqPrefix + "user.token"
```

Comment:
@felixgateru Why not we are not using same like operation `issue_token` and `refresh_token`?

Reply:
This was done to group the related events in a single stream